.gitignore

```diff
@@ -7,4 +7,7 @@
 .project
 .pydevproject
 wsgi/media/photologue/photos/
-wsgi/odyssy/.idea/
+wsgi/odyssy/
```

Comment:
but won't it will ignore all the files in that directory(ignoring the directory). Is there currently exists .idea folder in the current codebase??